### ISSUE TITLE
CLI to to data api Interactions

### ISSUE THREAD

**Issue by Jrecos** [Occlusion-Solutions/llm_explorer#7]
It would be nice to have a CLI app to interact with the data models and disengage visualization and code-engine

**Comment by Broomva** [comment]
Yes, it could leverage pydantic as the base structure of the API schemas and serve both as a cli interface (like $ llm_explorer --prompt '....') and the routes for a server
